### PR TITLE
Add attachment retrival option

### PR DESCRIPTION
**Pull request by EstebanSerfe**
This merge allows to have a function to retrieve the contents of the attachments for an specific email.

Open to suggestions or improvements.

### CODE REVIEW

**Comment by EstebanSerfe** [comment]
This was created on codeception 2.*

**Comment by Jamesking56** [comment]
Hi :wave: 

I'm now maintaining this project. Please can you take a look at @dzoeteman's review and let us know what your thoughts are.

Cheers.

File: src/Module/MailCatcher.php [hunk 1]
```diff
@@ -378,4 +378,29 @@ protected function grabMatchesFromEmail(Email $email, $regex)
...
+     */
+    public function grabAttachmentFromEmail($email, $index = 0) {
+        $this->assertNotNull($email);
+        // This is for version 2.1.x
+        if (is_array($email)) {
+            $this->assertArrayHasKey('attachments', $email);
+            $this->assertArrayHasKey($index, $email['attachments']);
+            return $this->mailcatcher->get($email['attachments'][$index]['href'])->getBody();
+        } else {
+            $response = $this->mailcatcher->get("/messages/{$id}.json");
```
[the remark this answers is not in the document]
Comment (EstebanSerfe) [reply src/Module/MailCatcher.php]: Hi, I din't set the function with the Last message idea as we had the testing case where multiple emails were sent and the one I'm interested on checking was not the last one, but the second previous one. This is why the method does not have a "lastEmail" section..

If you like I can get to review and add the method.

**Comment by Jamesking56** [comment]
This is conflicting, any chance of an update @EstebanSerfe?

**Comment by Jamesking56** [comment]
Superseded by #70

**Pull request closed without merging**